gh-badge.spec.js

```diff
@@ -0,0 +1,61 @@
+var assert = require('assert');
+var cproc = require('child_process');
```

Comment:
It's pretty confusing when imports are aliased like this. Since you're only using `spawn`, what about doing:

```
const {spawn} = require('child_process');
```